packages/mobile/src/screens/create-chat-blast-screen/ChatBlastSelectAudienceFields.tsx

```diff
@@ -0,0 +1,143 @@
+import { useGetCurrentUser } from '@audius/common/api'
+import { useField } from 'formik'
+
+import { Text } from '@audius/harmony-native'
+import { ExpandableRadio } from 'app/components/edit/ExpandableRadio'
+import { ExpandableRadioGroup } from 'app/components/edit/ExpandableRadioGroup'
+
+const TARGET_AUDIENCE_FIELD = 'target_audience'
+
+const messages = {
+  followers: {
+    label: 'My Followers',
+    description: 'Send a bulk message to all of your followers.'
+  },
+  supporters: {
+    label: 'Tip Supporters',
+    description: 'Send a bulk message to everyone who has tipped you.'
+  },
+  purchasers: {
+    label: 'Past Purchasers',
+    description:
+      'Send a bulk message to everyone who has purchased content from you on Audius.',
+    placeholder: 'Premium Content'
+  },
+  remixCreators: {
+    label: 'Remix Creators',
+    description:
+      'Send a bulk message to creators who have remixed your tracks.',
+    placeholder: 'Tracks with Remixes'
+  }
+}
+
+export const ChatBlastSelectAudienceFields = () => {
+  const [{ value }, , { setValue }] = useField(TARGET_AUDIENCE_FIELD)
+  return (
+    <ExpandableRadioGroup value={value} onValueChange={setValue}>
+      <FollowersMessageField />
+      <TipSupportersMessageField />
+      <PastPurchasersMessageField />
+      <RemixCreatorsMessageField />
+    </ExpandableRadioGroup>
+  )
+}
+
+const LabelWithCount = (props: {
+  label: string
+  count: number
+  isSelected: boolean
+}) => {
+  const { label, count, isSelected } = props
+  return (
+    <Text>
+      {label}
+      {isSelected ? <Text color='subdued'> ({count})</Text> : null}
+    </Text>
+  )
+}
+
+const FollowersMessageField = () => {
+  const { data: user } = useGetCurrentUser()
+  const { follower_count: followerCount } = user ?? { follower_count: 0 }
```

Comment:
nit: i'm not a fan of the `?? xxx`, what if we did `const followerCount = user.follower_count` and let it be `undefined` and return early if we need it later?